whois-nrtm4/src/main/java/net/ripe/db/nrtm4/DeltaChange.java

```diff
@@ -16,46 +18,54 @@ enum Action {
         DELETE,
         ADD_MODIFY;
 
-        @Override
         @JsonValue
-        public String toString() {
+        public String toLowerCaseName() {
             return name().toLowerCase();
         }
     }
 
+    @JsonIgnore
+    private final int serialId; // Only set if action is ADD_MODIFY
     private final Action action;
```

Comment:
it's used to track an object. the story said we should be able to trace an object in a snapshot back to whois. it was discussed with Ed -- he asked for the serial to be stored.

Reply:
Ok, but if it's ignored, it will not be persisted in database when you serialize it.  Or you are saving it in a different column in database?

Reply:
OK now I see it, you are persisting in snapshot_object table. But if its for snapshot only can we just pass the serial_id to function synchronizeDeltasToSnapshot like we pass version_id ? Instead of putting a json ignore on deltaChange as it is not used in delta change table and not to be serialized

Reply:
it's saved in snapshot_object so we can see which serial_id caused an object to appear in a snapshot. we'll also be able to use that to see which version it appeared in, if we ever need that.